sc_preprocessing/0_qc_notebooks/scanpy_analysis_synovium_bulk.ipynb

```diff
@@ -0,0 +1,2142 @@
+{
```

Comment:
<p>These and the ones after are commented out</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_analysis/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>removed in <a href="https://github.com/greenelab/buddi_analysis/pull/12/commits/390b86aebf04f040aeb6d49cfc62a7ae77301677" target="_blank">390b86a</a></p>